scripts/ec_transition

```diff
@@ -125,7 +125,7 @@ usage() {
 
 ROOT_DIR="/"
 
-[ -z "$BOARD_VENDOR" ] && error_exit "BOARD_VENDOR not given"
+[ -z "$SYSTEM_VENDOR" ] && error_exit "BOARD_VENDOR not given"
```

Comment:
Change the help text as well?

Reply:
Amended here f921869fc12670429fe1ec65f81eb9b3253b9462.